.gitignore

```
@@ -1,2 +1,4 @@
-.venv
-__pycache__
+.env
+.venv/
+/site/
+/.cache/
```

I'll be adding this later anyway
```suggestion
/.cache/
/.idea/
```